decred/tests/unit/util/test_database.py

```diff
@@ -49,11 +50,14 @@ def test_database(self):
             master = database.KeyValueDatabase(os.path.join(tempDir, "tmp.sqlite"))
 
             # '$' in bucket name is illegal.
-            self.assertRaises(Exception, lambda: master.child("a$b"))
+            self.assertRaises(ValueError, lambda: master.child("a$b"))
 
             try:
                 db = master.child("test")
 
+                # '$' in bucket name is illegal.
+                self.assertRaises(ValueError, lambda: db.child("c$d"))
```

Comment:
There will be two of these if you rebase.

Reply:
Not sure what you mean, I rebased and I don't see another one of these.

Unless you're referring to the almost identical check a few lines above, but that one exercises the code in the `KeyValueDatabase` class, while this one is for the code in the `Bucket` class.